_schema.graphql

```diff
@@ -4416,6 +4427,9 @@ type Mutation {
   # Fulfills an Order with one fulfillment by setting this fulfillment to all line items of this order
   fulfillOrderAtOnce(input: FulfillOrderAtOnceInput!): FulfillOrderAtOncePayload
 
+  # Confrims pickup for an ecommerce order
+  confirmPickup(input: ConfirmPickupInput!): ConfirmPickupPayload
```

Comment:
Please drop this one, I know it'll be inconsistent in clients (and I'd like to go and migrate everything to use `ecommerce` asap)